library/log_test.go

```diff
@@ -54,6 +54,8 @@ func TestLibrary_Log_MaskData(t *testing.T) {
 	s3Masked := "$ echo $SECRET1\n***\n$ echo $SECRET2\n***\n"
 	s4 := "SOME_SECRET=((%.YY245***pP.><@@}}"
 	s4Masked := "SOME_SECRET=***"
+	s5 := "www.post-secrets.com?username=secret&password=extrasecret"
+	s5Masked := "www.post-secrets.com?username=***&password=***"
```

Comment:
```suggestion
	s5 := "www.example.com?username=secret&password=extrasecret"
	s5Masked := "www.example.com?username=***&password=***"
```

suggestion: use reserved url intended for examples vs random url (see https://datatracker.ietf.org/doc/html/rfc2606#section-3)